samples.http

```diff
@@ -0,0 +1,13 @@
+### GET all entries
+GET http://localhost:3000/api/entries HTTP/1.1
```

Comment:
The default server port is `3001`:

```suggestion
GET http://localhost:3001/api/entries HTTP/1.1
```

But it's possible to change the `PORT` environment variable.
In this case, the `samples.http` would not target the correct server.

You can take a look at the REST Client documentation:
https://marketplace.visualstudio.com/items?itemName=humao.rest-client

An example:
https://raw.githubusercontent.com/lmachens/aeternum-map/main/samples.http
See first three lines.